CI failures

CI seems to have been intermittently failing for a long time. See all the red X's in

https://github.com/mewmew/float/commits/master

From https://travis-ci.org/mewmew/float/jobs/628377883?utm_source=github_status

`go test` pass and takes about 1 minute

`go test -race` times out after 10 minutes (there are a lot of test cases)

> No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

Besides this issue there is also a lint issue, but the above one is the main reason of the CI failures.

```
### golint

float128ppc/float128ppc.go:19:2: comment on exported var NegNaN should be of the form "NegNaN ..."

float128ppc/float128ppc.go:21:2: comment on exported var NaN should be of the form "NaN ..."
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -19,8 +19,8 @@ before_script:
   - wget https://github.com/mewmew/ci/raw/master/get_tools.sh
   - chmod +x get_tools.sh
   - ./get_tools.sh
-  - wget https://github.com/mewmew/ci/raw/master/ci_checks.sh
+  - wget https://github.com/pwaller/ci/raw/pwaller-patch-1/ci_checks.sh
   - chmod +x ci_checks.sh
 
 script:
-  - ./ci_checks.sh
+  - GOTEST_RACE=0 ./ci_checks.sh

